src/pages/home/components/ScrollTopButton.js

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import Button from 'components/button/Button';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faArrowUp } from '@fortawesome/free-solid-svg-icons';
+
+class ScrollTopButton extends React.Component {
+  state = { scrollTopClass: '' };
+
+  componentDidMount() {
+    this.scrollHandler();
+    window.addEventListener('scroll', this.scrollHandler);
+  }
+
+  componentWillUnmount() {
+    window.removeEventListener('scroll', this.scrollHandler);
+  }
+
+  scrollHandler = () => {
+    this.setState({
+      scrollToTopClass: window.pageYOffset > window.innerHeight / 4 ? 'opacity-100' : '',
+    });
+  };
+
+  clickHandler = () => {
+    window.scrollTo({
+      behavior: 'smooth',
+      top: 0,
+    });
+  };
+
+  render() {
+    const styles = {
```

Comment:
Since the values of this variable are 'static', what do you think about creating this `const styles` outside the component or as a `static property`?

Doing so, we would avoid the reinitialization every time it re-renders

Reply:
Chenges done.